impl/src/planning/subplanners.rs

```diff
@@ -143,40 +134,44 @@ impl<'planner> WorkspaceSubplanner<'planner> {
       workspace_path: self.settings.workspace_path.clone(),
       gen_workspace_prefix: self.settings.gen_workspace_prefix.clone(),
       output_buildfile_suffix: self.settings.output_buildfile_suffix.clone(),
+      workspace_members: self
+        .metadata
+        .metadata
+        .workspace_members
+        .iter()
+        .filter_map(|pkg_id| {
```

Comment:
I might be missing something but why not

```rust
self
.metadata
.metadata // <-- Also why this stutter?
.packages
.iter()
.find(|pkg| pkg.id == &pkg_id) // <-- Also why the javaesq `.eq(`
.map(|pkg| get_workspace_member_path(&pkg.manifest_path, &self.metadata.metadata.workspace_root))
```

Reply:
No, this needs to iterate workspace members specifically.

The 'stutter' is is the name `metadata` being used in two different contexts. One is the glob of metadata output by the metadata fetcher and the inner member is the `cargo_metadata::Metadata`.